x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/index.tsx

```diff
@@ -209,11 +210,18 @@ export function Detail() {
             </FlexItemWithMaxHeight>
             <EuiFlexItem>
               <EuiFlexGroup alignItems="center" gutterSize="m">
-                <FlexItemWithMinWidth grow={false}>
-                  <EuiText>
-                    {/* Render space in place of package name while package info loads to prevent layout from jumping around */}
-                    <h1>{integrationInfo?.title || packageInfo?.title || '\u00A0'}</h1>
-                  </EuiText>
+                <FlexItemWithMinWidth grow={true}>
+                  <EuiFlexGroup alignItems="center">
+                    <EuiFlexItem grow={false}>
+                      <EuiText>
+                        {/* Render space in place of package name while package info loads to prevent layout from jumping around */}
+                        <h1>{integrationInfo?.title || packageInfo?.title || '\u00A0'}</h1>
+                      </EuiText>
+                    </EuiFlexItem>
+                    <EuiFlexItem grow={false}>
+                      <EuiBadge color="default">Elastic Agent</EuiBadge>
```

Comment:
Should this text be translated?

Reply:
good point. I can wrap it in i18n-translation. Can always default to a no-op-translation. cc @mostlyjason any thoughts?

Reply:
I'm not familiar with our standard i18n policy, but allowing it to be translated seems like a good thing